### PR TITLE
Implement form group

### PR DESCRIPTION
**Pull request by lvoliveira**

### CODE REVIEW

File: src/ng-xform/fields-group/fields-group.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<div *ngFor="let field of fields">
+    <div [ngSwitch]="field.controlType">
+
+        <ng-template [ngSwitchCase]="'GROUP'">
+            <hr *ngIf='field.label'>
```
Comment (igortg) [inline src/ng-xform/fields-group/fields-group.component.html]: Don't you need to close this tag?

File: src/ng-xform/fields-group/fields-group.component.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <div [ngSwitch]="field.controlType">
+
+        <ng-template [ngSwitchCase]="'GROUP'">
+            <hr *ngIf='field.label'>
+            <h3 *ngIf='field.label'>
+                {{field.label}}
+            </h3>
+            <div [formGroup]='form'>
+                <div [formGroupName]="field.key">
+                    <ng-xform-fields-group [fields]='field.fields' [form]='form.get(field.key)' [editing]='editing'></ng-xform-fields-group>
```
Comment (igortg) [inline src/ng-xform/fields-group/fields-group.component.html]: Too long line

File: src/ng-xform/fields/field-group.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { DynamicField } from './dynamic-field';
+
+export class FieldGroup extends DynamicField {
```
Comment (igortg) [inline src/ng-xform/fields/field-group.ts]: `FieldGroup` is a way to edit a nested object. `NestedObjectField` would be more correct

**Review by igortg — CHANGES_REQUESTED** [review]

File: src/ng-xform/fields-group/fields-group.component.html (hunk 1 above)
Comment (lvoliveira) [reply src/ng-xform/fields-group/fields-group.component.html]: No, it is a horizontal line, it does not have content

**Review by edetec — APPROVED** [review]

**Comment by lvoliveira** [comment]
@igortg done

**Review by igortg — APPROVED** [review]

**Comment by lvoliveira** [comment]
@edetec , @igortg , could you review the last commit? It is a bug fix for a nested object with null value.

**Review by edetec — APPROVED** [review]
nice

**Pull request merged**